src/hotspot/share/gc/shared/cardTable.cpp

```diff
@@ -78,8 +77,8 @@ CardTable::CardTable(MemRegion whole_heap) :
 void CardTable::initialize(void* region0_start, void* region1_start) {
   size_t num_cards = cards_required(_whole_heap.word_size());
 
-  // each card takes 1 byte; + 1 for the guard card
-  size_t num_bytes = num_cards + 1;
+  // each card takes 1 byte
+  size_t num_bytes = num_cards;
```

Comment:
Instead of the comment, actually multiplying with the number of cards with the size of a `CardValue` seems preferable:
```suggestion
  size_t num_bytes = num_cards * sizeof(CardValue);
```

Reply:
Does the comment `each card takes 1 byte` need to be adjusted?

Reply:
just remove it imo. The code makes it clear that we calculate cards * sizeof(card) here.

Reply:
Fixed.